.github/workflows/main.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Cypress run
         uses: cypress-io/github-action@v5
         with:
-            spec: cypress/e2e/updateCards.cy.js
+            spec: cypress/e2e/attachOnCard.cy.js
```

Comment:
Update this to run the whole suite, you are only testing one test file, while on CICD all the tests should run